packages/cli/test/tsconfig.json

```diff
@@ -2,8 +2,12 @@
   "declaration": false,
   "extends": "../tsconfig",
   "compilerOptions": {
+    "target": "ESNext",
```

Comment:
Ah, I left that in by accident. It was changed because I wasn't able to set breakpoints on any generators. Omitting this causes every file to have an absurd number of "helpers" injected in by TS which slows down compilation and makes setting breakpoints difficult.